src/gateway/kong-manager/get-started/consumers.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Authentication with Consumers
+badge: free
+---
+
+In this example, you’re going to enable the **Key Authentication plugin**, then create a consumer that uses key authentication. API key authentication is one of the most popular ways to conduct API authentication and can be implemented to create and delete access keys as required.
+
+If you prefer to use the Admin API, check out the [{{site.base_gateway}} getting started guide](/gateway/latest/get-started/secure-services/).
+
+## Set up the Key Authentication Plugin
+
+On the Workspaces tab in Kong Manager:
+
+1. Open the **default** workspace.
+2. From the menu, open **Routes** and select the **mocking** route you created.
+4. From the sub-menu, select the **Plugins** tab, then click **Install Plugin**.
+5. Find the **Key Authentication** plugin and click **Enable**.
+6. On the **Install plugin: key-auth** page, the plugin fields are automatically scoped to the route because the plugin is selected from the mocking Routes page.
```

Comment:
```suggestion
6. On the **Install plugin: key-auth** page, the plugin fields are automatically scoped to the route because the plugin is selected from the mocking routes page.
```